mentors/java/s.json

```
@@ -38,5 +38,13 @@
   "link_url": "https://github.com/shankarshastri/",
   "avatar_url": "https://avatars0.githubusercontent.com/u/10391044",
   "bio": "Lover Of Pure Functional Programming."
+},
+{
+  "github_username": "sushovankarmakar",
+  "name": "Sushovan Karmakar",
+  "link_text": "GitHub Profile",
+  "link_url": "https://github.com/sushovankarmakar",
+  "avatar_url": null,
+  "bio": "I am professional Java Developer who want to contribute in open source. I want to share my knowledge and experiences to help others to reach their goals"
```

```suggestion
  },
  {
    "github_username": "sushovankarmakar",
    "name": "Sushovan Karmakar",
    "link_text": "GitHub Profile",
    "link_url": "https://github.com/sushovankarmakar",
    "avatar_url": null,
    "bio": "I am professional Java Developer who want to contribute in open source. I want to share my knowledge and experiences to help others to reach their goals"
```
The square brackets at the very top and very bottom of the file should not be indented, and they are not.  The parenthesis `{}` that surround each person's bio should be indented two spaces from the square brackets.  The contents of each bio should be indented 2 spaces (so a total of 4 spaces) from the `{}`.  My suggestions fixes the content of this file that you changed.